docs/get-started.md

```diff
@@ -4,14 +4,43 @@ title: Temporal documentation
 sidebar_label: Get started
 ---
 
+Welcome to Temporal's user documentation!
+
 Temporal reimagines state-dependent service-orchestrated application development, and so much more.
```

Comment:
I know this was already here but I want to say it better.

Reply:
I wrote rewrote this line from a number of sources. It wasn't copied or carried over from someplace.
Improving it, I think that is the hardest part of our job right now :) Not sure how to capture the essence in just 1-2 sentences yet. Unless you have an alternative suggestion, to keep things succinct I am going to push for keeping this as-is for now - because in its current form it is at least accurate and all encompassing - "so much more" :)